tests/webview/integration/test_rex_redirects.py

```diff
@@ -27,28 +27,36 @@ def test_archive_is_still_reachable(archive_base_url, rex_base_url):
         assert rex_base_url not in hist.headers["location"]
 
 
+@markers.webview
 @markers.test_case("C553080")
-@markers.rex
 @markers.nondestructive
-def test_redirecting_to_rex_from_within_webview(webview_base_url, rex_base_url, selenium):
-    """Webview needs to redirect to REX when one of the featured books is a REX book.
-    https://github.com/openstax/cnx/issues/401
-    """
+def test_redirecting_to_rex_from_within_webview(
+    webview_base_url, selenium, rex_base_url, rex_released_books
+):
+
+    # GIVEN the home page and REX released books list
+    for allbooks in rex_released_books:
+
+        home = Home(selenium, webview_base_url).open()
+
+        for book in home.featured_books.openstax_list:
+
+            # WHEN we click on a featured book
+            if book.cnx_id in allbooks:
+
+                book_title = book.title
```

Comment:
Is the `for allbooks in rex_released_books` even necessary? You are already using an if statement in the for loop. Could you use that better?

Reply:
This is to prevent stale element exception by resetting to webpage as it changes during the test. If I remove that, I am getting that exception.